File: javacountries/src/main/java/com/example/javacountries/JavacountriesApplication.java
package com.example.javacountries;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class JavacountriesApplication {

    //extanciate
    static CountryList myCountryList;

    public static void main(String[] args) {

        myCountryList = new CountryList();

        SpringApplication.run(JavacountriesApplication.class, args);
    }



}

File: src/main/kotlin/com/jakeesveld/java_countries/JavaCountriesApplication.kt
package com.jakeesveld.java_countries

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class JavaCountriesApplication

public var countryList: CountryList = CountryList()

fun main(args: Array<String>) {
    runApplication<JavaCountriesApplication>(*args)
}
